src/Mvc/Mvc.Razor/src/RazorPageOfT.cs

```diff
@@ -15,12 +15,11 @@ public abstract class RazorPage<TModel> : RazorPage
     /// <summary>
     /// Gets the Model property of the <see cref="ViewData"/> property.
     /// </summary>
-    public TModel? Model => ViewData == null ? default(TModel) : ViewData.Model;
```

Comment:
I like this style better :)
`public TModel? Model => ViewData?.Model ?? default(TModel);`

Reply:
We're no longer null-coalescing `ViewData.Model` so your suggestion wouldn't apply here.